syntax = "proto3";

message tiltmsg {
    enum colour_type {
        UNKNOWN = 0;
        RED = 1;
        GREEN = 2;
        BLACK = 3;
        PURPLE = 4;
        ORANGE = 5;
        BLUE = 6;
        YELLOW = 7;
        PINK = 8;
      }
    uint32 version = 1;
    colour_type colour = 2;
    uint32 temperature = 3;
    uint32 specificGravity = 4;
    uint32 timeStamp = 5;
    uint32 txPower = 6;
}